uptime/types.go

```diff
@@ -0,0 +1,72 @@
+package uptime
+
+import (
+	"fmt"
+)
+
+// TODO: correct types?
+type PeerID = string;
+type MultiAddr = string;
```

Comment:
Use multiaddress type

Reply:
All the above types are also tied to the actor code as well. I'll create a follow up issue to change all of them together